Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,82 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
```

Comment:
Good job showing your calculations for each snippets runtime Bryce, and a really nice and succinct breakdown for exercise # 2 👍